### ISSUE TITLE
Function Request: The ability to command units when remote controling a squad leader

### ISSUE THREAD

**Issue by ReactionAndy** [Asaayu/integrated-voice-control-system#2]
Is it possible to make it so you can command units when remote controling an AI squad leader in zeus.

**Comment by Asaayu** [comment]
I think this is just related to my use of the `player` command rather then a `remote unit/player` version. Should be an easy fix but I don't have an ETA at the moment.

**Comment by Asaayu** [comment]
I've added support for remote controlled units in the latest update, do you want to give it a go and make sure it works for you?

**Comment by ReactionAndy** [comment]
The mod seems to freeze now when I use the Hotkey. It shows the green border but wont disappear after releasing the hotkey. Also it does not pick up anything I say.
It worked before the update and I've tried to Repair the mod but it still freezes.

The game seems to run normaly otherwise. it's just that the mod does not work.

I also tried disabling other mods. except only with the voice integrated and CBA mod.

The IVCS Audio test does not pick up anything I say either.
My michrophone does work in game.

I've tried reinstalling the mod but no luck.

The latest update I'm running is the 3:33 PM CEST 8/26/2021 - Steam Workshop
